client-hc/src/test/java/com/graphhopper/api/AbstractGHMatrixWebTester.java

```diff
@@ -185,17 +188,19 @@ public void testReadingWeights_TimesAndDistances() throws IOException {
     @Test
     public void noVehicleWhenNotSpecified() {
```

Comment:
We should probably change this test to check there is no **profile** if not specified now.